src/components/SocialMedia.jsx

```diff
@@ -0,0 +1,21 @@
+import { Social } from "./Social"
```

Comment:
since you've created an index file that exports all components you can import your components from there for consistency